packages/ui/src/Toggle.module.scss

```diff
@@ -0,0 +1,98 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+$h: 24px;
```

Comment:
@utkukaratas as you can see in other components, all custom sizes are multipliers of `c.$grid`. This way all of our components will scale if we ever decide to change the grid. Moreover, `c.$grid` is in `rem`. So our UI scales for visually impaired users who increase their base font size